sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyAlert/GetAlertsOptions.cs

```diff
@@ -19,22 +25,30 @@ public GetAlertsOptions(DateTimeOffset startTime, DateTimeOffset endTime, TimeMo
         }
 
         /// <summary>
+        /// Filters the result. Only alerts triggered after this point in time, in UTC, will be returned.
         /// </summary>
         public DateTimeOffset StartTime { get; }
 
         /// <summary>
+        /// Filters the result. Only alerts triggered before this point in time, in UTC, will be returned.
         /// </summary>
         public DateTimeOffset EndTime { get; }
 
         /// <summary>
+        /// Specifies to which time property of an <see cref="AlertResult"/> the filters <see cref="StartTime"/>
+        /// and <see cref="EndTime"/> will be applied.
         /// </summary>
         public TimeMode TimeMode { get; }
 
         /// <summary>
+        /// If set, skips the first set of items returned. This property specifies the amount of items to
+        /// be skipped.
         /// </summary>
         public int? SkipCount { get; set; }
 
         /// <summary>
+        /// If set, specifies the maximum limit of items returned by each service call. Note: all items will
+        /// returned, even if more service calls are required.
```

Comment:
The concept of pageable with a `top` operator that is actually only limiting page size is really confusing for users. I suggest something like this:
```suggestion
        /// If set, specifies the maximum limit of items returned in each page of results. Note: 
        /// unless the number of pages enumerated from the service is limited, the service will 
        /// return an unlimited number of total items.
```